### DISCUSSION TITLE
Portuguese 3K

### DISCUSSION THREAD

**Discussion by michelkoga** [monkeytypegame/monkeytype#1306] [Questions] [answered]
Hi, I just created a Portuguese 3K list, and was wondering if it could be added.
The current Portuguese 200 list is too basic. My 3K is ideal for Portuguese students as well as for typing. 
If possible, I would like to make the pull request, just to learn how to do it. If not possible, I can just send it from here. 

Thanks in advance

**Comment by unrealapex** [comment, marked as answer]
It can be added, just make sure to submit a PR with the `_list.json` and `_groups.json` files modified in incorporate the new file added along with a `portuguese_3k.json` file.

**Reply by michelkoga** [reply]
Got it! Thank you very much!
